platformio/platformio.bzl

```diff
@@ -210,6 +210,9 @@ def _emit_build_action(ctx, project_dir):
       env={
         # The PlatformIO binary assumes that the build tools are in the path.
         "PATH":"/bin:/usr/bin:/usr/local/bin:/usr/sbin:/sbin",
+        "LC_ALL":"C.UTF-8",
```

Comment:
Would you mind adding a comment above these new envs explaining why we need them and maybe including a link to the bug?